prow/test/integration/prow/config.yaml

```diff
@@ -16,3 +16,24 @@ prowjob_default_entries:
     repo: 'testRepo'
     config:
       tenant_id: 'tester'
+
+gerrit:
+  tick_interval: 30s
+  org_repos_config:
+  - org: http://fakegerritserver
+    repos:
+    - gerrit-test-infra
+
+plank:
+  default_decoration_config_entries:
+  - config:
+      timeout: 2h
+      grace_period: 15s
+      utility_images:
+        clonerefs: gcr.io/k8s-prow/clonerefs:v20220517-17b68df070
```

Comment:
In https://github.com/kubernetes/test-infra/pull/26394 I define these utility images inside the integration tests's `.prow-images.yaml` so that I can refer to them as `localhost:5001/clonerefs:latest` from the ProwJob Spec. This way any changes to clonerefs will be caught in the same test run (instead of referring to a static clonerefs images as you've done here). I think you can do the same thing here, because doing so will at least speed up performance when running the integration test locally (no need to fetch this image from GCR when you can refer to the locally-built one).